src/net/sourceforge/plantuml/klimt/creole/SheetBlock1.java

```diff
@@ -140,13 +140,16 @@ private void initMap(StringBounder stringBounder) {
 
 			sea.doAlign();
 			sea.translateMinYto(y);
-			sea.exportAllPositions(positions);
 			final double width = sea.getWidth();
 			widths.put(stripe, width);
 			minMax = sea.update(minMax);
 			final double height = sea.getHeight();
 			heights.put(stripe, height);
 			y += height;
+			if (stringBounder.matchesProperty("TIKZ")) {
+				sea.doAlignTikzBaseline();
+			}
+			sea.exportAllPositions(positions);
```

Comment:
sea.update, and sea.getHeight would re-calculate the height of positions. However, we only want to change the translation without the actual height, so move the adjust at last.